app/controllers/shipit/stacks_controller.rb

```diff
@@ -39,6 +39,11 @@ def destroy
     end
 
     def settings
+      client = Shipit::ApiClient.find_by(creator: current_user, name: "#{current_user.name} ccmenu client")
+      client ||= Shipit::ApiClient.create!(
```

Comment:
Ah thank you, putting permissions in the find_or_create_by was getting me `SerializationTypeMismatch`, couldn't figure out the foo :)